### PR TITLE
Commerce 5582 An error is thrown after complete the checkout with a Virtual Product

### PR DESCRIPTION
**Pull request by jaydawu**
https://issues.liferay.com/browse/COMMERCE-5582
Resent from https://github.com/liferay-commerce/liferay-portal/pull/3557

### CODE REVIEW

**Comment by jaydawu** [comment]
ci:test:sf

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceVirtualProducts.testcase [hunk 1]
```diff
@@ -15,6 +15,141 @@ definition {
...
+				Click(
+					key_contentType = "Base Information",
+					locator1 = "CommerceEntry#NOTIFICATION_DELIVERY_PANEL_CLOSED");
+			}
+
+			Select(
+				key_fieldLabel = "Activation Status",
+				locator1 = "Select#GENERIC_SELECT_FIELD",
+				value1 = "Pending");
+		}
```
Comment (pietrobocale) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceVirtualProducts.testcase:84]: Can you remove all these steps? For this issue, we don't need to configure anything. We just need to create the product

Thanks!

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceVirtualProducts.testcase [hunk 2]
```diff
@@ -15,6 +15,141 @@ definition {
...
+
+		task ("And completes a checkout after adding the virtual product to the cart no console errors are thrown") {
```
Comment (pietrobocale) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceVirtualProducts.testcase:86]: Can we add the login with the buyer before we add the product to the cart?

Thanks!

**Review by pietrobocale — CHANGES_REQUESTED** [review]

**Comment by pietrobocale** [comment]
ci:test:commerce-functional

**Comment by jaydawu** [comment]
ci:test:sf

**Comment by jaydawu** [comment]
ci:test:sf

**Comment by jaydawu** [comment]
ci:test:relevant

**Comment by jaydawu** [comment]
ci:test:commerce-functional

**Comment by jaydawu** [comment]
ci:test:relevant

**Review by pietrobocale — APPROVED** [review]

**Comment by pietrobocale** [comment]
Review Passed!

**Comment by jaydawu** [comment]
ci:test:relevant

**Comment by jaydawu** [comment]
ci:test:commerce-functional

**Comment by jaydawu** [comment]
I will forward this since no errors related to this pull.

**Comment by jaydawu** [comment]
ci:forward:force

**Comment by jaydawu** [comment]
Hello @andrea-ale-sbarra, could you please take a look at this pull? I'm really sorry that I force forward this, please help check if it's safe for forwarding it. Thanks in advance!

cc/ @victoroscuro

**Comment by victoroscuro** [comment]
@andrea-ale-sbarra I double-checked the results, usual failures + ViewPortalSmokeArchive7413 whose failure should not be caused by the PR changes. 
FYI @jaydawu

**Pull request closed without merging**